templates/windows/project.nuspec.erb

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id><%= @name %></id>
+    <version><%= "#{@version}.#{@release}" %></version>
+    <title><%= @name %></title>
+    <authors><%= @vendor.match(/^(.*) <.*>$/)[1] %></authors>
+    <owners><%= @vendor.match(/^(.*) <.*>$/)[1] %></owners>
+    <projectUrl><%= @homepage %></projectUrl>
+    <packageSourceUrl><%= "https://github.com/puppetlabs/#{project}-vanagon" %></packageSourceUrl>
+    <projectSourceUrl><%= "https://github.com/puppetlabs/#{project}" %></projectSourceUrl>
+    <docsUrl><%= @homepage %></docsUrl>
+    <mailingListUrl><%= @homepage %></mailingListUrl>
+    <bugTrackerUrl><%= @homepage %></bugTrackerUrl>
+    <licenseUrl><%= @homepage %></licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/puppet.svg</iconUrl>
+    <description>
+<%= @description %>
+    </description>
+    <summary><%= @description.lines.first.chomp %></summary>
+    <releaseNotes><%= @homepage %></releaseNotes>
+    <copyright><%= @vendor.match(/^(.*) <.*>$/)[1] %></copyright>
+    <tags>puppet configuration management infrastructure automation facter hiera mco mcollective marionette collective</tags>
+    <provides><%= get_replaces.map { |replace| "#{replace.replacement}" }.join(" ") %></provides>
+    <replaces><%= get_replaces.map { |replace| "#{replace.replacement}" }.join(" ") %></replaces>
+  </metadata>
+  <files>
+<%- unless @bill_of_materials -%>
+    <file src="../bill-of-materials" target="usr/share/doc" />
+<%- end -%>
+<%- (get_files + get_configfiles + get_directories).each do |file| -%>
+    <file src="<%= file.path %>" target="<%= File.dirname file.path.sub(/^\/cygdrive\/c/,'') %>" />
```

Comment:
Yeah, I have to define the target directories with `/cygdrive/c` because something about how erb is populating the makefile is appending my working directory to the path when I use `C:/`, which is frustrating. Because of that, I have to get rid of the reference to `cygdrive` here :( More investigation on this is required